### PR TITLE
571 Urological Supplies Progress Note

### PR DESCRIPTION
**Pull request by yunwwang**
PR for creating Urological Supplies Progress Note

### CODE REVIEW

File: UrologicalSupplies/R4/resources/Questionnaire-R4-UrologicalSuppliesProgressNote.json [hunk 1]
```diff
@@ -0,0 +1,1799 @@
...
+            "code": "N",
+            "display": "No"
+          }
+        ]
+      }
+    }
+  ],
+  "item": [
+    {
+      "linkId": "PBD",
```
Comment (yuliumitre) [inline UrologicalSupplies/R4/resources/Questionnaire-R4-UrologicalSuppliesProgressNote.json]: Do we already using this format for the linkId yet?

File: UrologicalSupplies/R4/resources/Questionnaire-R4-UrologicalSuppliesProgressNote.json [hunk 2]
```diff
@@ -0,0 +1,1799 @@
...
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"VentilatorsPrepopulation\".Today"
+              }
+            }
+          ],
+          "text": "Date of encounter (MM/DD/YYYY):",
```
Comment (yuliumitre) [inline UrologicalSupplies/R4/resources/Questionnaire-R4-UrologicalSuppliesProgressNote.json]: Date of the evaluation is in the Provider section. If you are using the substitution, that is fine with me.

File: UrologicalSupplies/R4/resources/Questionnaire-R4-UrologicalSuppliesProgressNote.json [hunk 3]
```diff
@@ -0,0 +1,1799 @@
...
+      "type": "group",
+      "item": [
+        {
+          "linkId": "DIAG.1",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"VentilatorsPrepopulation\".CurrentDiagnoses"
```
Comment (yuliumitre) [inline UrologicalSupplies/R4/resources/Questionnaire-R4-UrologicalSuppliesProgressNote.json]: On the form, I don't see this is populated. 
Screen Shot 2020-05-06 at 9 39 46 AM

File: UrologicalSupplies/R4/resources/Questionnaire-R4-UrologicalSuppliesProgressNote.json [hunk 4]
```diff
@@ -0,0 +1,1799 @@
...
+          ],
+          "text": "Patient Qualifying Diagnoses",
+          "type": "open-choice",
+          "repeats": true,
+          "required": true,
+          "answerOption": []
+        }
+      ]
+    },
+    {
```
Comment (yuliumitre) [inline UrologicalSupplies/R4/resources/Questionnaire-R4-UrologicalSuppliesProgressNote.json]: Are we omitting this section? 
Screen Shot 2020-05-06 at 9 42 50 AM

File: UrologicalSupplies/R4/resources/Questionnaire-R4-UrologicalSuppliesProgressNote.json [hunk 5]
```diff
@@ -0,0 +1,1799 @@
...
+          "text": "Visual",
+          "type": "text"
+        },
+        {
+          "linkId": "OBJ.11",
+          "text": "Other",
+          "type": "text"
+        }
+      ]
+    },
```
Comment (yuliumitre) [inline UrologicalSupplies/R4/resources/Questionnaire-R4-UrologicalSuppliesProgressNote.json:472]: Do we need this section and it is  in the template?
Screen Shot 2020-05-06 at 9 44 53 AM

Comment (yuliumitre) [reply UrologicalSupplies/R4/resources/Questionnaire-R4-UrologicalSuppliesProgressNote.json:472]: Oh, I see, you move it together with the treatment plan section.

File: UrologicalSupplies/R4/resources/Questionnaire-R4-UrologicalSuppliesProgressNote.json [hunk 6]
```diff
@@ -0,0 +1,1799 @@
...
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"BasicPractitionerInfoPrepopulation\".NPI"
+              }
+            }
+          ],
+          "text": "NPI",
+          "type": "string",
```
Comment (yuliumitre) [inline UrologicalSupplies/R4/resources/Questionnaire-R4-UrologicalSuppliesProgressNote.json]: Name and NPI are not populated. 

Screen Shot 2020-05-06 at 9 49 44 AM

File: UrologicalSupplies/R4/resources/Questionnaire-R4-UrologicalSuppliesProgressNote.json (hunk 1 above)
Comment (yunwwang) [reply UrologicalSupplies/R4/resources/Questionnaire-R4-UrologicalSuppliesProgressNote.json]: Changed back

File: UrologicalSupplies/R4/resources/Questionnaire-R4-UrologicalSuppliesProgressNote.json (hunk 3 above)
Comment (yunwwang) [reply UrologicalSupplies/R4/resources/Questionnaire-R4-UrologicalSuppliesProgressNote.json]: Copy/Past error. Fixed

**Pull request merged**